Bottom: 3bb2d5c4840cc0d59a5a500ba614c6001a92dc00
Top:    d3cf6446fd6282f1865b0aab7918b786da23eafb
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-11-15 22:46:17 -0800

Refresh of test-udp

---

diff --git a/src/preload.c b/src/preload.c
index 2d44848..01f0b0e 100644
--- a/src/preload.c
+++ b/src/preload.c
@@ -660,8 +660,8 @@ ssize_t recv(int socket, void *buf, size_t len, int flags)
 {
 	int fd;
 	struct sockaddr sa;
-	socklen_t len = sizeof sa;
-	real.recvfrom(socket, buf, len, flags, &sa, &len);
+	socklen_t slen = sizeof sa;
+	real.recvfrom(socket, buf, len, flags, &sa, &slen);
 	return (fd_fork_get(socket, &fd) == fd_rsocket) ?
 		rrecv(fd, buf, len, flags) : real.recv(fd, buf, len, flags);
 }
@@ -775,7 +775,7 @@ use_rpoll:
 		rfds[i].fd = fd_getd(fds[i].fd);
 		rfds[i].events = fds[i].events;
 		rfds[i].revents = 0;
-		real.poll(&fds[i], 1, timeout);
+		real.poll(&fds[i], 1, 0);
 	}
 
 	ret = rpoll(rfds, nfds, timeout);
